src/Main.java

```diff
@@ -29,24 +29,25 @@ public static void main(String[] args) {
             System.out.println(i);
         }
         System.out.println("Задача 8");
-        int Deposit = 29000;
-        int  monthlyDeposit = 0;
+        int deposit = 29000;
+        int monthlyDeposit = 0;
         for (int month = 1; month <= 12; month++) {
-            monthlyDeposit = Deposit/12;
-            System.out.println("Месяц " + Deposit + "Сумма накоплений равна" + monthlyDeposit + "рублей");
+            monthlyDeposit = deposit + monthlyDeposit;
+            System.out.println("Месяц " + month + "Сумма накоплений равна" + monthlyDeposit + "рублей");
         }
         System.out.println("Задача 9");
+        for (int month = 1; month <= 12; month++) {
+            deposit += (deposit * 0.12) / 12;
```

Comment:
должно быть как в предыдущей - каждый месяц откладываем + проценты
```suggestion
            monthlyDeposit = deposit + monthlyDeposit;
            monthlyDeposit += monthlyDeposit / 100; // 1% каждый месяц - это 12% годовых
```